pkg/runtime/runtime.go

```diff
@@ -1180,7 +1181,15 @@ func (a *DaprRuntime) publishMessageGRPC(msg *pubsub.NewMessage) error {
 		PubsubName:      msg.Metadata[pubsubName],
 	}
 
-	if data, ok := cloudEvent[pubsub.DataField]; ok && data != nil {
+	if data, ok := cloudEvent[pubsub.DataBase64Field]; ok && data != nil {
+		decoded, decode_err := base64.StdEncoding.DecodeString(data.(string))
+		if decode_err != nil {
+			log.Warnf("unable to base64 decode cloudEvent field data_base64: %s", decode_err)
```

Comment:
Can you use `log.Debugf` here? this is a per operation log message

Reply:
Sure, will fix.